src/lib.rs

```diff
@@ -110,9 +110,9 @@ macro_rules! cast_or_ret {
     }};
 }
 
-/// Thread-local instance of the Cryptographically Secure Pseudo-Random Number
-/// Generator (CSPRNG). This is used to avoid contention and locking between
-/// different threads.
+/* Thread-local instance of the Cryptographically Secure Pseudo-Random Number
+ * Generator (CSPRNG). This is used to avoid contention and locking between
+ * different threads. */
```

Comment:
I would probably add a note here with explanation why this is not a doc string, otherwise next one going through the missing docs string in the source will try to convert it back ;)

Reply:
I am not sure we'll go thorough that again, I think it is fine as is, it is easy to change even if we regress, it is only a cosmetic annoyance.
If we care really to no "regress" I would rather add a CI test that fails on these kind of warnings